js-conditions-and-loops/conditions.js

```diff
@@ -0,0 +1,40 @@
+//Using "if else"
+const averageGrade = 85;
+
+if (averageGrade <= 59){
+    console.log("Незадовільно");
+
+}else if (averageGrade>60&&averageGrade<=70){
```

Comment:
please add spaces for conditions. It is hard to read
`averageGrade > 60 && averageGrade <= 70`

Reply:
your previous condition is `<= 59` and this one `>60`. In the end value 60 is not handled at all

Reply:
it should be >=60

Reply:
same for next conditions

Reply:
fixed